example/eblob_backend.c

```diff
@@ -214,6 +278,16 @@ static int blob_write(struct eblob_backend_config *c, void *state,
 
 	memcpy(key.id, io->id, EBLOB_ID_SIZE);
 
+	/*
+	 * Check timestamp of the existing record if compare-and-swap timestamp flag is set.
+	 * If on-disk record's timestamp is higher than that in IO structure, do not allow this write.
+	 */
+	if (io->flags & DNET_IO_FLAGS_CAS_TIMESTAMP) {
```

Comment:
We do not have API to get neither file info nor `dnet_io_attr` from the backend.
To optimize things, backend sends requested data to the client directly, higher layer like `dnet_process_cmd_raw()` and IO threads can not get this info.